### PR TITLE
Remove outdated Makefiles

### PR DESCRIPTION
**Pull request by tyabus**

### CODE REVIEW

**Comment by FreeSlave** [comment]
If you're gonna remove makefiles, remove lines about them from README.md too.

**Comment by tyabus** [comment]
> If you're gonna remove makefiles, remove lines about them from README.md too.

Whoops, i forget about README.md

File: README.md [hunk 1]
```diff
@@ -45,17 +45,17 @@ TODO
...
 
 ### OS X
 
 Nothing here.
 
 ### FreeBSD
 
-    (cd dlls && gmake CXX=clang++ CC=clang)
-    (cd cl_dll && gmake CXX=clang++ CC=clang)
+    (CC=clang CXX=clang++ ./waf configure -T release)
```
Comment (nekonomicon) [inline README.md]: Any reason to use CC/CXX environment variables here?
And why not use one block:
### Linux/*BSD/OS X
or
### Unix-like

**Comment by nekonomicon** [comment]
Also, to run waf you need to install python.

**Pull request merged**